test/utils.sh

```diff
@@ -139,6 +147,19 @@ function deploy_test_env {
    echo "Verifying there are no (terminating) pods of type test-env"
    wait_for_it 600 "$cli get pods --namespace=$TEST_APP_NAMESPACE_NAME --selector app=test-env --no-headers | wc -l | tr -d ' ' | grep '^0$'"
 
+  conjur_node_name="conjur-cluster"
+  if [ "$CONJUR_DEPLOYMENT" = "dap" ]; then
+      conjur_node_name="conjur-follower"
+  fi
+  conjur_appliance_url=https://$conjur_node_name.$CONJUR_NAMESPACE_NAME.svc.cluster.local
```

Comment:
Please rename to more generic name that is appropriate to OSS as well. Say conjur_url